azure-pipelines.yml

```diff
@@ -33,11 +33,11 @@ steps:
     command: 'build'
     projects: 'HelpMyStreetFE/HelpMyStreetFE/HelpMyStreetFE.csproj'
 
-#- task: DotNetCoreCLI@2
-#  displayName: DotNetCore test
-#  inputs:
-#    command: 'test'
-#    arguments: '--configuration $(buildConfiguration) --collect "Code coverage"'
+- task: DotNetCoreCLI@2
+  displayName: DotNetCore test
+  inputs:
+    command: 'test'
```

Comment:
is this missing a "projects" tag?

Reply:
None of our other pipelines seem to need one, but I've tried adding it...

Reply:
Seems to have done the trick.